uwblueprint/community-fridge-kw
# fix: email reminders

## Brief description. What is this change? 

### Donor Email Reminders



## Implementation description. How did you make this change?
* Email Reminders are sent every 24hr at 12pm for donations that are scheduled between the next 24-48 hrs.
* An api route is exposed which sends email for donations that are scheduled between the next 24-48 hrs.
* Cron job is scheduled through: https://cron-job.org/ to hit the created endpoint every 24 hours at 12pm

This approach has some flaws because any body can hit that endpoint and send emails. We should be adding some sort of authorization layer to it.



## Steps to test
1. Create some donations
2. Keep an eye out on your mailbox for reminder emails. Ensure that you are not being spammed and you are only sent one email




## Checklist
